browser/brave_wallet/solana_provider_renderer_browsertest.cc

```diff
@@ -532,6 +536,15 @@ class SolanaProviderRendererTest : public InProcessBrowserTest {
 
   void SetUpOnMainThread() override {
     InProcessBrowserTest::SetUpOnMainThread();
+
+    auto* keyring_service =
+        brave_wallet::KeyringServiceFactory::GetServiceForContext(
+            browser()->profile());
+    // Create wallet since native wallet should not be injected otherwise
+    keyring_service->CreateWallet("password", base::DoNothing());
```

Comment:
we need a test suite to test wallet create of not affects if provider should be injected. And for other test suites, just change the prefs.

Reply:
Added additional tests for solana and eth